agent/docs/none-platform.md

```diff
@@ -1,21 +1,25 @@
-# None Platform Support
+# None and External Platform Support
```

Comment:
I would suggest renaming the file to none-and-external-plaform.md.

Reply:
Done